src/parser/type_checking.c

```diff
@@ -10915,6 +10891,13 @@ pt_common_type (PT_TYPE_ENUM arg1_type, PT_TYPE_ENUM arg2_type)
     {
       common_type = PT_TYPE_DOUBLE;
     }
+#if 0 //bSolo: why is this here? maybe wrong merge?
```

Comment:
@bsolomenco @paulgheorghecristian 

I don't think it is a mistake. Paul added it for a QA issue.

Guys, please have a chance to discuss it.

Reply:
I think there is a misunderstanding: I put the message there for myself to check it later because Remzi noticed the difference from develop branch and told me to investigate
As I can see now, in develop there is no equivalent code here, I'll remove the comment and the commented code (#if 0...#endif), it was a wrong on my side when I manually merged develop into my branch after a conflict in this file
I will also add Paul as reviewer

Reply:
@eseokoh 
According to CBRD-21674, we decided that JSON is compatible only with string. So this is why this code was removed by @cristighr1995 .

Reply:
ah. You are right. I forgot the fix :)